Packs/CommonScripts/Scripts/StixCreator/StixCreator.py

```diff
@@ -83,6 +85,45 @@
 }
 
 
+def search_relationship_indicators(value: str) -> list[dict]:
+    relationships = demisto.searchRelationships({"entities": [value]}).get("data", [])
+
+    query = ""
+    for rel in relationships:
+        entity_a = rel.get("entityA", "")
+        entity_b = rel.get("entityB", "")
+        if entity_a.lower() == value.lower():
+            query += f'"{entity_b}"'
+        else:
+            query += f'"{entity_a}"'
+        query += " or "
+    demisto_indicators = demisto.searchIndicators(query=query).get("iocs", [])
+    return demisto_indicators
+
+
+def get_indicators_stix_ids(value: str, indicator_type: str, demisto_indicators: list[dict]) -> list[str]:
+    stix_ids = []
+    for indicator in demisto_indicators:
+        if stix_id := indicator.get("stixid"):
+            stix_ids.append(stix_id)
+        else:
+            stix_type = XSOAR_TYPES_TO_STIX_SDO.get(indicator.get("indicator_type", ""))
+            if not stix_type:
+                stix_type = XSOAR_TYPES_TO_STIX_SCO.get(indicator.get("indicator_type", ""), 'indicator')
+            if stix_type == 'indicator':
```

Comment:
not sure what do we need this

Reply:
this was for debugging purposes, deleted